lib/command/src/Obelisk/Command.hs

```diff
@@ -154,6 +155,9 @@ deployCommand cfg = hsubparser $ mconcat
       , command "ios" $ info ((,) <$> pure IOS <*> fmap pure (strArgument (metavar "TEAMID" <> help "Your Team ID - found in the Apple developer portal"))) mempty
       ]
 
+    reboot :: Parser Bool
+    reboot = flag False True (long "reboot")
```

Comment:
Rebooting on deploy is currently the default behaviour, this shouldn't change that. Otherwise users may be surprised to see that their running apps aren't the latest version following a deployment.

Reply:
I'd argue that it should've never been turned on by default in the first place, This should've always been the users choice to *actually* reboot

Reply:
You're talking about restarting the backend service, right?

Reply:
I was mistaken this is referring to the restart of the machine the backend service is running on. The backend service will still be restarted as part of a deploy that did not include the `--reboot` flag.